tests/infra/ccf.py

```diff
@@ -300,6 +300,7 @@ def wait_for_node_commit_sync(self):
         for _ in range(3):
             commits = []
             for node in self.nodes:
+                LOG.info("Talking to node {}".format(node.node_id))
```

Comment:
That's redundant because the client logging contains the node id in the request line.